install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -147,6 +147,15 @@ containers:
 {{- end }}
 {{- if .Values.global.trustDomain }}
   - --trust-domain={{ .Values.global.trustDomain }}
+{{- end }}
```

Comment:
I don't think 'sds enabled' can be a global option - please add explicit code in agent to detect if  the token exists, and which kind of token it is.

Reply:
k8s sa jwt token exist in every pod, while trustworthy jwt isn't; today we use flags to decide which token to use (to request cert/key)